### PR TITLE
Removed 2VM and other irrelevant tests from nightly20220314.

### PR DESCRIPTION
**Pull request by chandrasekhard2**

### CODE REVIEW

File: tests/tensorflow/nightly20220314/classifier-efficientnet.libsonnet [hunk 1]
```diff
@@ -171,9 +171,9 @@ local tpus = import 'templates/tpus.libsonnet';
     efficientnet + v2_32 + functional + tpuVm,
     //efficientnet + v3_32 + functional,
     //efficientnet + v3_32 + convergence + timeouts.Hours(24),
-    efficientnet + v4_8 + functional + tpuVm,
-    efficientnet + v4_32 + functional + tpuVm,
-    efficientnet + v4_8 + convergence + tpuVm,
-    efficientnet + v4_32 + convergence + tpuVm,
+    //efficientnet + v4_8 + functional + tpuVm,
```
Comment (will-cromar) [inline tests/tensorflow/nightly20220314/classifier-efficientnet.libsonnet:174]: Did you mean to remove these TPU VM configurations?

**Review by will-cromar — COMMENTED** [review]
Can you also rebase onto master?

File: tests/tensorflow/nightly20220314/classifier-efficientnet.libsonnet (hunk 1 above)
Comment (chandrasekhard2) [reply tests/tensorflow/nightly20220314/classifier-efficientnet.libsonnet:174]: Yes. I commented them out.

Comment (will-cromar) [reply tests/tensorflow/nightly20220314/classifier-efficientnet.libsonnet:174]: I see -- you're removing the v4 tests.

File: tests/tensorflow/nightly20220314/nlp-wmt.libsonnet [hunk 2]
```diff
@@ -55,12 +55,12 @@ local utils = import 'templates/utils.libsonnet';
     accelerator: tpus.v3_32,
```
Comment (will-cromar) [inline tests/tensorflow/nightly20220314/nlp-wmt.libsonnet:55]: Can you remove these files that have everything in `configs` commented out? Remember to also remove them from targets.jsonnet, or the code won't compile.

**Review by will-cromar — CHANGES_REQUESTED** [review]

File: tests/tensorflow/nightly20220314/nlp-wmt.libsonnet (hunk 2 above)
Comment (chandrasekhard2) [reply tests/tensorflow/nightly20220314/nlp-wmt.libsonnet]: Done! Deleted files with zero configs.

**Review by will-cromar — APPROVED** [review]

**Pull request merged**